systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -151,4 +154,75 @@ public static void waitForClusterStability(String clusterName) {
+     * @param clusterName name of the cluster where Kafka resource can be found
+     * @param kafkaDynamicConfiguration enum instance, which defines all supported configurations
+     * @param value value of specific property
+     */
+    public static void updateConfigurationWithStabilityWait(String clusterName, KafkaDynamicConfiguration kafkaDynamicConfiguration, Object value) {
+        updateSpecificConfiguration(clusterName, kafkaDynamicConfiguration, value);
+        PodUtils.verifyThatRunningPodsAreStable(KafkaResources.kafkaStatefulSetName(clusterName));
+    }
+
+    /**
+     * Method, verifying that updating configuration were successfully changed inside Kafka CR
+     * @param kafkaDynamicConfiguration enum instance, which defines all supported configurations
+     * @param value value of specific property
+     */
+    public static boolean verifyCrDynamicConfiguration(String clusterName, KafkaDynamicConfiguration kafkaDynamicConfiguration, Object value) {
+        LOGGER.info("Dynamic Configuration in Kafka CR is {}={} and excepted is {}={}",
+            kafkaDynamicConfiguration.toString(),
+            KafkaResource.kafkaClient().inNamespace(kubeClient().getNamespace()).withName(clusterName).get().getSpec().getKafka().getConfig().get(kafkaDynamicConfiguration.toString()),
+            kafkaDynamicConfiguration.toString(),
+            value);
+
+        return KafkaResource.kafkaClient().inNamespace(kubeClient().getNamespace()).withName(clusterName).get().getSpec().getKafka().getConfig().get(kafkaDynamicConfiguration.toString()).equals(value);
+    }
+
+    /**
+     * Method, which, verifying that updating configuration were successfully changed inside Kafka pods
+     * @param kafkaPodNamePrefix prefix of Kafka pods
+     * @param kafkaDynamicConfiguration enum instance, which defines all supported configurations
+     * @param value value of specific property
+     * @return
+     * true = if specific property match the excepted property
+     * false = if specific property doesn't match the excepted property
+     */
+    public static boolean verifyPodDynamicConfiguration(String kafkaPodNamePrefix, KafkaDynamicConfiguration kafkaDynamicConfiguration, Object value) {
+
+        List<Pod> kafkaPods = kubeClient().listPodsByPrefixInName(kafkaPodNamePrefix);
+
+        for (Pod pod : kafkaPods) {
+
+            String result = cmdKubeClient().execInPod(pod.getMetadata().getName(), "/bin/bash", "-c", "bin/kafka-configs.sh --bootstrap-server localhost:9092 --entity-type brokers --entity-name 0 --describe").out();
+
+            LOGGER.debug("This dyn.configuration {} inside the Kafka pod {}", result, pod.getMetadata().getName());
+
+            if (!result.contains(kafkaDynamicConfiguration + "=" + value)) {
+                LOGGER.error("Kafka Pod {} doesn't contain {} with value {}", pod.getMetadata().getName(), kafkaDynamicConfiguration.toString(), value);
```

Comment:
Maybe we should show dyn.configuration in case of error as well?

Reply:
?

Reply:
If the error occurs then we show the `Kafka Pod {}....`